### ISSUE TITLE
AdsCommon::Errors::TypeMismatchError

### ISSUE THREAD

**Issue by kaushikvinaytg** [googleads/google-api-ads-ruby#75]
Somehow I seem to get this error 
AdsCommon::Errors::TypeMismatchError: AdsCommon::Errors::TypeMismatchError: expected: 'choice subtype', provided: 'ExpressionRuleUserList' for field '{:id=>185402532}' 
when trying to create LogicalUserList using adwords_api

Code

```ruby
adwords = AdwordsApi::Api.new
user_list_srv = adwords.service(:AdwordsUserListService, :v201509)
operation = {
          :operator => 'ADD',
          :operand => {
            :xsi_type => 'LogicalUserList',
            :name => "Retargeting list",
            :description => "Custom combination list",
            :status => 'OPEN',
            :membership_life_span => '30',
            :rules => [
              {
                :operator => 'ANY',
                :rule_operands => [
                  {
                    :xsi_type => "ExpressionRuleUserList",
                    :id => 185402532
                  },
                  {
                    :xsi_type => "ExpressionRuleUserList",
                    :id => 185402652
                  },
                  {
                    :xsi_type => "ExpressionRuleUserList",
                    :id => 185402772
                  }
                ]
              }
            ]
          }
        }
response = user_list_srv.mutate([operation])
```

**Comment by mcloonan** [comment]
I believe I've narrowed down the issue here. When the registry is generated, the :choices field for this use case is only populated with one item, UserList, because that's the only one explicitly mentioned in the WSDL. However, subclasses of UserList should also be explicitly allowed here, so the fix should be to fix registry generation and then regenerate the registry for the AdwordsUserListService.

**Comment by kaushikvinaytg** [comment]
That makes sense, I was only able to pass UserList as xsi_type for any other subtype it threw an error like i mentioned in previous comment.

**Comment by dklimkin** [comment]
Should be fixed in 37efcfdd287187a930d91276e36b67b658a58f25.
